kola/tests/coretest/cloudinit.go

```diff
@@ -0,0 +1,125 @@
+package coretest
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+	"syscall"
+)
+
+const cloudinitBinPath = "/usr/bin/coreos-cloudinit"
+
+func read(filename string) (string, error) {
+	bytes, err := ioutil.ReadFile(filename)
+	return string(bytes), err
+}
+
+func rmdir(path string) error {
+	cmd := exec.Command("sudo", "rm", "-rf", path)
```

Comment:
Sounds reasonable but this file was a directly taken from github.com/coreos/coretest. The only thing I changed when porting coretest files was to change the function signatures of the tests to match that expected by the remote function runner. Then I added some glue to group the tests into 3 different kola tests that each call a series of native functions.

If coretest needs some reorganizing maybe we can do another PR?